### PR TITLE
Optimise images in attributed and comparisons

### PR DESCRIPTION
**Pull request by rwd**

### CODE REVIEW

File: components/generic/CompareImageSlider.vue [hunk 1]
```diff
@@ -108,6 +135,7 @@
 
     data() {
       return {
+        maxWidth: 1100,
```
Comment (mirjamverloop) [inline components/generic/CompareImageSlider.vue]: Where is this used?

File: components/generic/CompareImageSlider.vue [hunk 2]
```diff
@@ -150,7 +178,8 @@
 
     methods: {
       setImageWidth() {
-        this.imageWidth = this.$refs.rightImage.getBoundingClientRect().width;
+        console.log(this.$refs.rightImage);
```
Comment (mirjamverloop) [inline components/generic/CompareImageSlider.vue]: Remove the console.log.

File: components/generic/OptimisedImage.vue [hunk 3]
```diff
@@ -0,0 +1,100 @@
+<template>
+  <span>
```
Comment (mirjamverloop) [inline components/generic/OptimisedImage.vue]: Is the span necessary? It breaks the image compare. If possible remove it here, or else, the image compare code should be slightly updated.  

```diff --git a/components/generic/CompareImageSlider.vue b/components/generic/CompareImageSlider.vue
index 77221612..a0eda8d4 100644
--- a/components/generic/CompareImageSlider.vue
+++ b/components/generic/CompareImageSlider.vue
@@ -246,7 +246,7 @@
     position: relative;
     overflow: hidden;
 
-    img {
+    span {
       width: 100%;
 
       &:nth-child(1) {
```

File: components/generic/OptimisedImage.vue [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    <b-img-lazy
+      v-if="lazy"
+      :src="optimisedSrc"
+      :width="optimisedWidth"
+      :height="optimisedHeight"
```
Comment (mirjamverloop) [inline components/generic/OptimisedImage.vue]: Could you set this to blank-width and blank-height, so CSS can still do some resizing if needed.

File: components/generic/CompareImageSlider.vue (hunk 1 above)
Comment (rwd) [reply components/generic/CompareImageSlider.vue]: Nowhere.

File: components/generic/OptimisedImage.vue (hunk 3 above)
Comment (rwd) [reply components/generic/OptimisedImage.vue]: It seems not. I will remove it.

**Pull request merged**